### PR TITLE
Small typo and formatting fixes

### PR DESCRIPTION
**Pull request by Qwerty-Space**
For example, this:
Before image
Becomes this:
After image

And this:
Default
Becomes this:
Default woo

### CODE REVIEW

File: lua/plugins/onmessage.lua [hunk 1]
```diff
@@ -176,9 +176,9 @@ function plugin.onEveryMessage(msg)
 							api.sendReply(msg, message, 'html')
 						elseif media_status == 'del' and n + 1 >= max then
 							api.deleteMessage(msg.chat.id, msg.message_id)
-							local message =
-							i18n([[%s, this type of media is <b>not allowed</b> in this chat.\n<i>The next time you will be banned/kicked/muted</i>
-								]]):format(name)
+							local message = i18n('
```
Comment (yangm97) [inline lua/plugins/onmessage.lua]: `'` is not valid for multi-line strings, please use `[[ ]]`.

Comment (yangm97) [inline lua/plugins/onmessage.lua]: ' is not valid for multi-line strings, please use [[ ]].

**Comment by Qwerty-Space** [comment]
I will update this PR with changes to the rest of the file, and other files too.

File: lua/plugins/onmessage.lua [hunk 2]
```diff
@@ -176,9 +176,8 @@ function plugin.onEveryMessage(msg)
 							api.sendReply(msg, message, 'html')
 						elseif media_status == 'del' and n + 1 >= max then
 							api.deleteMessage(msg.chat.id, msg.message_id)
-							local message =
-							i18n([[%s, this type of media is <b>not allowed</b> in this chat.\n<i>The next time you will be banned/kicked/muted</i>
-								]]):format(name)
+							local message = i18n([[%s, this type of media is <b>not allowed</b> in this chat.
+							<i>Next time you will be banned, kicked, or muted.</i>]]):format(name)
```
Comment (TiagoDanin) [inline lua/plugins/onmessage.lua]: Remove the space at the beginning of the line

**Comment by yangm97** [comment]
I guess we could break this PR into smaller PRs? My priority list would be:

1. Fix broken multi-line\nstrings asap
2. Move strings to the top of files to make reading code easier (some files already do that, each with its own way,  let’s try to make it consistent this time), fix code folding, make redundant strings pop up and make it easier to do...
3. English QA and add context for translators.

I would not change any strings in phase 2 (so gettext notices strings just moved), leaving that for phase 3, and of course, phase 2 and 3 could be split into multiple PRs too.

**Comment by Qwerty-Space** [comment]
But not on my machine because git messed them up

**Comment by yangm97** [comment]
I think I did 1. but not sure if it fixed everything btw

**Comment by yangm97** [comment]
Can we close this then?

**Comment by Qwerty-Space** [comment]
@yangm97 Your choice.

**Pull request closed without merging**